glean-core/android/src/main/java/mozilla/telemetry/glean/debug/GleanDebugActivity.kt

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.debug
+
+import android.app.Activity
+import android.os.Bundle
+import mozilla.telemetry.glean.Glean
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * Debugging activity exported by Glean to allow easier debugging.
+ * For example, invoking debug mode in the Glean sample application
+ * can be done via adb using the following command:
+ *
+ * adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean.debug.GleanDebugActivity
```

Comment:
Might want to update this, the package name is no longer `mozilla.components.service...`

Reply:
Thank you very much!